File: examples/php/src/Todo.php
<?php

namespace TodoFilter;

/**
 * Todo model
 * Represents a single todo item
 */
class Todo
{
    public int $id;
    public string $title;
    public string $status;
    public \DateTime $createdAt;

    public function __construct(
        int $id,
        string $title,
        string $status,
        ?\DateTime $createdAt = null
    ) {
        $this->id = $id;
        $this->title = $title;
        $this->status = $status;
        $this->createdAt = $createdAt ?? new \DateTime();
    }
}

File: demos/TodoFilter/src/TodoFilter/Models/Todo.cs
namespace TodoFilter.Models;

/// <summary>
/// Represents a todo item
/// </summary>
public class Todo
{
    public int Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Status { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}
